pkg/manager-server/file_system_gfs2.go

```diff
@@ -104,6 +105,27 @@ func (f *FileSystemGfs2) Create(devices []string, opts FileSystemOptions) error
 		return err
 	}
 
+	userID := 0
+	if _, exists := opts[UserID]; exists {
+		userID = opts[UserID].(int)
+	}
+
+	groupID := 0
+	if _, exists := opts[GroupID]; exists {
+		groupID = opts[GroupID].(int)
+	}
+
+	mountpath := "/mnt/nnf/client/" + f.name
+	if err := f.Mount(mountpath); err != nil {
+		return err
+	}
+
+	defer func() { err = f.Unmount(mountpath) }()
```

Comment:
Won't this setting of 'err' replace any error value from the os.Chown?

Reply:
Yeah. Good point I'll change it to check for an existing error value first.